system-addon/content-src/components/AutoMigratePrompt/AutoMigratePrompt.jsx

```diff
@@ -0,0 +1,32 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage, injectIntl} = require("react-intl");
+const {actionTypes: at, actionCreators: ac} = require("common/Actions.jsm");
+
+const AutoMigratePrompt = props => {
+  if (!props.AutoMigrate.display) {
+    return null;
+  }
+
+  return props.AutoMigrate.stage === 0 ?
+    (<div className="migrate-prompt">
+      <span>{props.AutoMigrate.msg}</span>
+      <div className="confirm">
+        <button onClick={props.onUndoClick}><FormattedMessage id="migrate_undo_button" /></button>
+        <button className="primary" onClick={props.onImportClick}><FormattedMessage id="migrate_import_button" /></button>
+      </div>
+    </div>) : (<div className="migrate-prompt">
+      <FormattedMessage id="migrate_manual_import_msg" />&nbsp;
+      <FormattedMessage id="migrate_manual_import_link_msg" />&nbsp;
```

Comment:
I see that AutoMigrate will be passing over at least one of the messages. Any reason why it shouldn't pass over the manual import messages too? I do see that parts of it will be linked, but multiple message parts could be passed over too.